Makefile.am

```diff
@@ -17,6 +17,9 @@ pm_font__DATA = fonts/Vera.ttf fonts/VeraMono.ttf
 pm_shaders__DATA = src/libprojectM/Renderer/blur.cg \
                    src/libprojectM/Renderer/projectM.cg
 
+# system headers/libraries/data to install
+include_HEADERS = config.h
```

Comment:
It's preferable to not need to install config.h:
https://sourceware.org/autobook/autobook/autobook_66.html
https://stackoverflow.com/questions/19586211/correct-installation-of-config-h-for-shared-library-using-autotools?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa